src/app/page/Home/Home.js

```diff
@@ -83,52 +362,41 @@ const HomePage = ({ auth, history }) => {
     <Page isLoaded={isLoaded} LoadingComponent={LoadingComponent}>
       {isEmpty ? (
         <Grid container>
+          <SignInHeaderComponent history={history} />
+          <PoweredByComponent />
           <Grid container>
-            <img src={HomeImage1} className={classes.HomeImage} alt="" />
-          </Grid>
-          <Grid container>
-            <Button
-              className={classes.SigninButton}
-              onClick={() => history.push("/login")}
-              data-testid="btn-login"
-            >
-              Sign In
-            </Button>
-          </Grid>
-          <Grid container justify="center">
-            <Body1>Dont have an account yet?</Body1>
-          </Grid>
-          <Grid container>
-            <Button
-              variant="outlined"
-              className={classes.SignupButton}
-              onClick={() => history.push("/signup")}
-              data-testid="btn-signup"
-            >
-              Sign Up
-            </Button>
+            <H4 className={classes.QuickInfoLabel}>
+              We're a grassroots team in Studio City, CA getting fresh farm produce to our neighbors
+              in need.
+            </H4>
           </Grid>
-          <Card>
-            <Grid container>
-              <H2 className={classes.Paragraph}>How it works</H2>
-              <Grid container>
-                <Body1>Need help?</Body1>
-                <Body1 className={classes.Paragraph}>
-                  Sign up to make a request for a food box, small errand, or a pharmacy pickup. Your
-                  ask goes right to the local coordinator, and is matched with a volunteer who will
-                  take care of your need ASAP.
-                </Body1>
-                <Body1>Want to help?</Body1>
-                <Body1 className={classes.Paragraph}>
-                  Sign up to join your local network of volunteers helping their neighbors through
-                  this crisis. Deliver food, medicine, and other supplies to the most vulnerable.
-                </Body1>
-              </Grid>
-              <Grid container>
-                <img src={HomeImage2} className={classes.HomeImage} alt="" />
-              </Grid>
-            </Grid>
-          </Card>
+          <GreetingCardComponent
+            title="Need help?"
+            message="Sign up to request a food box, small errand, or a pharmacy pickup. You'll be matched with a
+            volunteer who will take care of you ASAP."
+            actions={[
+              {
+                label: "I Need Help",
+                onClick: () => null,
+              },
+            ]}
```

Comment:
I dont think the pattern you've created here is the right direction for this.
We should use React APIs where available because it seems like you're re-creating something React already does for us.

The abstraction with a greetingCardComponent seems like the right direction. But I didn't see any actions props with more than one object in them.

This might be better served just passing a prop for a linkUrl and linkLabel if each card only has one call to action.

Reply:
Addressed in f8a481f. Removed the actions prop and simplified it to actionLabel, actionPress.